packages/eslint-plugin-office-addins/README.md

```diff
@@ -0,0 +1,52 @@
+# eslint-plugin-office-addins
+
+eslint plugin for office-addins
+
+## Installation
+
+You'll first need to install [ESLint](http://eslint.org):
+
+```
+$ npm i eslint --save-dev
+```
+
+Next, install `eslint-plugin-office-addins`:
+
+```
+$ npm install eslint-plugin-office-addins --save-dev
+```
+
+**Note:** If you installed ESLint globally (using the `-g` flag) then you must also install `eslint-plugin-office-addins` globally.
+
+## Usage
+
+Add `office-addins` to the plugins section of your `.eslintrc` configuration file. You can omit the `eslint-plugin-` prefix:
+
+```json
+{
+    "plugins": [
+        "office-addins"
+    ]
+}
+```
+
+Then configure the extends property.
+```json
+{
+    "extended": [
```

Comment:
This doesn't look right. You say "extends" property but it says "extended" in the JSON.  Shouldn't there just also only be one plugin? I think it could be more clear about just choosing one (recommended), and choosing a different one if React or React Native is being used.